kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md

```diff
@@ -384,3 +384,34 @@ Sometimes in production, but most likely in testing environments, you might want
 $ kubectl create -f delete-domain-job.yaml
 
 ```
+## Troubleshooting
+
+1. Message: "status on iteration 20 of 20
+pod domain1-create-weblogic-sample-domain-job-4qwt2 status is Pending
+The create domain job is not showing status completed after waiting 300 seconds."  
+* The most likely cause is related to the value of persistentVolumeClaimName, defined in domain-home-on-pv/create-domain-inputs.yaml. Change the value to weblogic-sample-pvc.
```

Comment:
This should detail how to determine if there's a mount issue.   E.g. how to get the events for the pod/job (via kubectl describe) and how to interpret the event that indicates the pod/job can't find a PVC/PV.    This should also mention how to fix the problem by modifying a value directly in the domain resource yaml (in case the user wants to modify just the domain resource without needing to run the whole 'create-domain' script again)...

Reply:
what is the domain resource yaml? Does the user need to just edit and save?

Reply:
The domain resource is the key resource for handling a domain in this project.   Sometimes we call it domain custom resource.  This an extremely key central tenet in the operator project - if it's not clear I suggest reading up on teh project's background documents in the site document.

We have a problem with customers conflating the sample's input flie with the domain resource -- as the sample input file + create-domain.sh script is in essence just a 'throw away' that sets up a domain resource file for you, that you can then customize.   The script also optionally calls kubectl create to deploy the resource...

Reply:
ok. read through https://github.com/oracle/weblogic-kubernetes-operator/blob/master/site/domain-resource.md

Not clear what to do after editing the domain resource... do we just need to call kubectl apply again?

Reply:
so... create-weblogic-domain/domain-home-on-pv/create-domain.sh generates three yaml files:
```
Generating /ddrive/vkraemer/gener8d-yamls//weblogic-domains/domain1/create-domain-job.yaml
Generating /ddrive/vkraemer/gener8d-yamls//weblogic-domains/domain1/delete-domain-job.yaml
Generating /ddrive/vkraemer/gener8d-yamls//weblogic-domains/domain1/domain.yaml
```
I am assuming that domain.yaml is the one to edit and save?

Do I need to do anything else after that?

Reply:
OK.  Stepping back a bit, it looks like we need to (a) recommend the cleanup steps (delete-domain script), and (b) fixing the inputs, and (c)

Reply:
looks like I needed to do 
```
kubectl apply -f  /ddrive/vkraemer/gener8d-yamls//weblogic-domains/domain1/domain.yaml
```
after I did the edit... is that correct???

Reply:
I've reconsidered my suggestion to work with the domain yaml after seeing how much of 'unknown' the domain yaml is at this point in the sample - instead we should mention some sort of cleanup step (to bring down the hurting domain) and steps a-d above.      The cleanup steps are described later in the sample (at least they are in the quick-start).

In this case, the cleanup step could be 'kubectl delete -f ...domainyaml...'

Reply:
made changes to address these comments